src/components/molecules/UserList/UserList.tsx

```diff
@@ -31,7 +31,7 @@ interface PropsType {
 }
 
 const UserList: React.FunctionComponent<PropsType> = ({
-  users,
+  users: userList,
```

Comment:
[NB, nit] I usually use the convention of `_VARNAME` (eg. `_users`) when 'shadowing' a variable like this. I find it helps me to know that I shouldn't really be using that variable 'deeper' within the component, and then I set whatever override/fix I need to it in the first couple of lines of the component.

Alternatively, we could be more semantice with the name here? eg. `allUsers`, and then further down `const visibleUsers` or similar? Or even just keep this as `users` and then call the other `visibleUsers`?